courses/free-courses-en.md

```diff
@@ -768,6 +769,11 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Julia
+
+* [Beginner Julia Tutorial](https://www.youtube.com/playlist?list=PLP8iPy9hna6SCcFv3FvY_qjAmtTsNYHQE) - The Julia Programming Language (YouTube)
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
It should be fixed now